matsim/src/main/java/org/matsim/core/mobsim/qsim/qnetsimengine/flow_efficiency/FlowEfficiencyCalculator.java

```diff
@@ -17,10 +17,10 @@
      *
      * @param qVehicle         the vehicle that consumes efficiency
      * @param previousQVehicle the vehicle previously traveling on this link before {@code qVehicle}, may be null
-     * @param previousTimeDiff time difference of the previous vehicle traveling on this link. Can be null if two vehicles are in the buffer
-     *                         simultaneously.
+     * @param previousTimeDiff time gap to the previous vehicle traveling on this link. Can be null if two vehicles are in the buffer
```

Comment:
Is this still correct? The only time when it will be null is when there is no previous veh, or are there still other cases?

Reply:
No you are right. In fact, it will only be null for the first vehicle ever passing the link..